Sming/Makefile

```diff
@@ -177,6 +177,12 @@ ENABLE_ESPCONN ?= 0
 ifeq ($(ENABLE_CUSTOM_LWIP), 1)
 	THIRD_PARTY_DATA += third-party/esp-open-lwip/Makefile.open
 	EXTRA_INCDIR += third-party/esp-open-lwip/include
+	EXTRA_CFLAGS_LWIP = -I../../system/include -I../../Wiring
+	ENABLE_LWIPDEBUG ?= 1
```

Comment:
Can you set the default for ENABLE_LWIPDEBUG to 0? Otherwise LWIP may start printing a lot of data.